modules/i3s/src/lib/parsers/parse-slpk/cd-file-header.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * Class for handling central directory file header of zip file
```

Comment:
Is this a zip file parser? Let's put it in a separate directory and have names include `zip`